### PR TITLE
Lecture1

### PR DESCRIPTION
**Pull request by NestorRnD**
My homework. Please check it.

### CODE REVIEW

File: Lecture_1/Exercise_5/calc_lim.c [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+
+int main(void){
+    uint64_t n=0,i;
+    
+    n--;
+    for(i=2;n>0;i++)
+	n/=i;
+    printf("%d\n",i);
+    return 0;
+}
```
Comment (hell-coder) [inline Lecture_1/Exercise_5/calc_lim.c:13]: Looks like the file is not  from here)

**Review by hell-coder — APPROVED** [review]
Everything is fine.
the only information note is:
Try to follow Linux coding style (missing spaces, redundant new lines..)

File: Lecture_1/Exercise_5/calc_lim.c (hunk 1 above)
Comment (NestorRnD) [reply Lecture_1/Exercise_5/calc_lim.c:13]: This file i use for calculate max possible argument for factorial function. ;-)

**Comment by NestorRnD** [comment]
Ok! Thanks for review.
I close this pull-request.
P.S. I try to follow Linux coding style in next homework (from lecture 5).

**Comment by NestorRnD** [comment]
Should i merge this branch with master?

**Pull request merged**